tasks/reverse_proxy.yml

```diff
@@ -3,6 +3,11 @@
   docker_volume:
     name: "{{ volume_names.tls_certificates }}"
 
+- name: Create a custom folder that can contain additional configuration for traefik
+  file:
+    path: "{{ traefik_extra_config_path }}"
```

Comment:
the way you defined it in the defaults, it should be `traefik.extra_config_path` (object notation)